### PR TITLE
RHDEVDOCS-4229 Release Notes Pipelines 1.7.2

### PR DESCRIPTION
**Pull request by sounix000**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.9`, `enterprise-4.10`, `enterprise-4.11`
- **JIRA issues**: [RHDEVDOCS-4229 Release Notes Pipelines 1.7.2](https://issues.redhat.com/browse/RHDEVDOCS-4229)
- **Preview pages**: Unavailable due to issues with Netlify. However, you can download this [generated HTML page](https://drive.google.com/file/d/13Q-dQUA4xZ6AYf4SsIL-eO286G9a8QtW/view?usp=sharing) and view it using your browser.
- **SME review**: @concaf       
- **Peer-review**: @Preeticp     

Manual cherry-pick for `enterprise-4.9`: https://github.com/openshift/openshift-docs/pull/47893

### CODE REVIEW

File: modules/op-release-notes-1-7.adoc [hunk 1]
```diff
@@ -362,3 +362,51 @@ With this update, {pipelines-title} General Availability (GA) 1.7.1 is available
...
+* Before this update, all S2I build tasks failed with an error similar to the following message:
++
+[source,terminal]
+----
+Error: error writing "0 0 4294967295\n" to /proc/22/uid_map: write /proc/22/uid_map: operation not permitted
+time="2022-03-04T09:47:57Z" level=error msg="error writing \"0 0 4294967295\\n\" to /proc/22/uid_map: write /proc/22/uid_map: operation not permitted"
+time="2022-03-04T09:47:57Z" level=error msg="(unable to determine exit status)"
+----
++
+With this update, the `pipelines-scc` security ontext constraint (SCC) allows the `SETFCAP` capability necessary for S2I and Buildah clustertasks. As a result, the S2I build tasks run successfully.
```
Comment (concaf) [inline modules/op-release-notes-1-7.adoc:411]:
```suggestion
With this update, the `pipelines-scc` security context constraint (SCC) allows the `SETFCAP` capability necessary for S2I and Buildah clustertasks. As a result, the S2I build tasks run successfully.
```

**Review by concaf — APPROVED** [review]

**Review by Preeticp — COMMENTED** [review]
@sounix000 some minor nits, PTAL

File: modules/op-release-notes-1-7.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (sounix000) [reply modules/op-release-notes-1-7.adoc:411]: `setfcap`, `setpcap`, etc. are various capabilities. Its usage is correct.

**Pull request merged**